File: app/src/main/java/com/mobile/rxjava2andretrofit2/mine/view/IMineDetailsView.java
package com.mobile.rxjava2andretrofit2.mine.view;

import com.mobile.rxjava2andretrofit2.base.IBaseView;
import com.mobile.rxjava2andretrofit2.first_page.bean.FirstPageDetailsResponse;

import java.util.List;

public interface IMineDetailsView extends IBaseView {

    void mineDetailsSuccess(List<FirstPageDetailsResponse.DataBean> success);

    void mineDetailsError(String error);
}

File: app/src/main/java/com/mobile/rxjava2andretrofit2/kotlin/mine/view/IMineDetailsView.kt
package com.mobile.rxjava2andretrofit2.kotlin.mine.view

import com.mobile.rxjava2andretrofit2.base.IBaseView
import com.mobile.rxjava2andretrofit2.kotlin.mine.bean.Data

interface IMineDetailsView : IBaseView {

    fun mineDetailsSuccess(success: List<Data>);

    fun mineDetailsError(error: String?);
}